liwords-ui/src/gameroom/board_panel.tsx

```diff
@@ -125,6 +130,44 @@ export const BoardPanel = React.memo((props: Props) => {
     };
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
+  useEffect(() => {}, [props.board]);
```

Comment:
what's this for?

Reply:
Might be a merge error, might be something I forgot I started. Removing.